udata_croquemort/checker.py

```diff
@@ -24,13 +24,22 @@ def _format_response(self, response):
             check_date = response.get('updated')
             if check_date:
                 check_date = dateutil.parser.parse(response.get('updated'))
-            return {
+
+            result = {
                 'check:url': response.get('checked-url'),
                 'check:status': status,
                 'check:available': status and status >= 200 and status < 400,
                 'check:date': check_date,
             }
 
+            for header in [
+                'content-type', 'content-length', 'content-md5', 'charset',
+                'content-disposition'
+            ]:
+                result[f"check:headers:{header}"] = response.get(header)
```

Comment:
maybe add a safe guard here? `if in` or `.get`

Reply:
🙈

Reply:
`.get` already has a safeguard, it's not a direct key access with `dict[key]`
```
Python 3.7.7 (default, Mar 10 2020, 15:43:33) 
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> print({}.get('a'))
None
```